lib/functions/installs.bash

```diff
@@ -25,7 +25,17 @@ install_command() {
 }
 
 get_concurrency() {
-  if command -v nproc >/dev/null 2>&1; then
+  if [ -n "$ASDF_CONCURRENCY" ]; then
+    printf "%s\n" "$ASDF_CONCURRENCY"
+    return
+  fi
+
+  local asdf_concurrency=
+  asdf_concurrency=$(get_asdf_config_value 'concurrency')
```

Comment:
Thinking on this further, `auto` might be the better default value so that we rely on `nproc` and co and only when the user explicitly sets `ASDF_CONCURRENCY` or `concurrency` will we use what they provide.

Reply:
I like the `auto` special value - agree its good to make it the default